locales/en-US/app.ftl

```diff
@@ -720,6 +720,16 @@ TransformNavigator--collapse-direct-recursion = Collapse recursion: { $item }
 #   $item (String) - Name of the function that transform applied to.
 TransformNavigator--collapse-function-subtree = Collapse subtree: { $item }
 
+## Source code view in a box at the bottom of the UI.
+
+SourceView--loading-url = Waiting for { $host }…
```

Comment:
Although they might seem obvious, given their names, but all these strings need comments for the variable (see strings before for the format)

Reply:
Oh, of course. Sorry about that.